dataplane/linux/int_dataplane.go

```diff
@@ -1263,11 +1263,35 @@ func (d *InternalDataplane) setUpIptablesBPF() {
 			rulesConfig.RouteSource == "WorkloadIPs" {
```

Comment:
Comment above is a bit out of date; you're now marking more packets than it says.

Reply:
Thanks, I've updated it now.